diff --git a/lua/barbar/events.lua b/lua/barbar/events.lua
index c5da3b5..3236b3c 100644
--- a/lua/barbar/events.lua
+++ b/lua/barbar/events.lua
@@ -439,6 +439,8 @@ function events.main_click_handler(bufnr, _, btn, _)
   -- NOTE: in Vimscript this was not `==`, it was a regex compare `=~`
   if btn == 'm' then
     bdelete(false, bufnr)
+  elseif btn == 'r' then
+    vim.cmd("vertical sbuffer " .. bufnr)
   else
     render.set_current_win_listed_buffer()
     set_current_buf(bufnr)
